### PR TITLE
Add uk.co.mrbenshef.Boop-GTK

### PR DESCRIPTION
**Pull request by mrbenshef**

### CODE REVIEW

File: data/uk.co.mrbenshef.Boop-GTK.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Boop-GTK
```
Comment (bilelmoussaoui) [inline data/uk.co.mrbenshef.Boop-GTK.desktop:2]: This should be part of the source tree, it's not a flatpak/flathub specific thing.

File: data/uk.co.mrbenshef.Boop-GTK.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Comment=Port of IvanMathy's Boop to GTK, a scriptable scratchpad for developers.
+Type=Application
+Exec=boop-gtk
+Terminal=false
+Categories=Utility;GTK;
+Keywords=Rust;Flatpak;GTK;Tools;
```
Comment (bilelmoussaoui) [inline data/uk.co.mrbenshef.Boop-GTK.desktop:8]:
```suggestion
Keywords=GTK;Tools;
```
not very useful and could mislead the user's research

File: data/uk.co.mrbenshef.Boop-GTK.desktop [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+Icon=uk.co.mrbenshef.Boop-GTK.png
```
Comment (bilelmoussaoui) [inline data/uk.co.mrbenshef.Boop-GTK.desktop:9]:
```suggestion
Icon=uk.co.mrbenshef.Boop-GTK
```

File: data/uk.co.mrbenshef.Boop-GTK.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bilelmoussaoui) [inline data/uk.co.mrbenshef.Boop-GTK.metainfo.xml:1]: Should be part of the upstream tree as well

**Comment by bilelmoussaoui** [comment]
bot, build uk.co.mrbenshef.Boop-GTK

File: uk.co.mrbenshef.Boop-GTK.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            "build-commands": [
+                "ls -al target/release/gn_out/obj",
+                "cargo --offline build --release",
+                "install -Dm755 ./target/release/boop-gtk -t /app/bin/",
+                "install -Dm644 ./flatpak/data/${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/",
+                "install -Dm644 ./flatpak/data/${FLATPAK_ID}.desktop -t /app/share/applications/",
+                "install -Dm644 ./flatpak/data/${FLATPAK_ID}.png -t /app/share/icons/hicolor/256x256/apps/"
+            ],
+            "sources": [
+                {
```
Comment (bilelmoussaoui) [inline uk.co.mrbenshef.Boop-GTK.json:47]: This shouldn't be a dir but rather a git source/tarball

**Comment by mrbenshef** [comment]
bot, build uk.co.mrbenshef.Boop-GTK

**Comment by mrbenshef** [comment]
bot, build uk.co.mrbenshef.Boop-GTK

**Comment by bilelmoussaoui** [comment]
Looks good to me, the app looks great!
/merge

**Comment by mrbenshef** [comment]
Thanks @bilelmoussaoui!

**Pull request closed without merging**